### PR TITLE
feat: add view tax group

### PR DESCRIPTION
**Pull request by BLasan**
## Description
View Tax Group component created.

## Related issues and discussion
#587 

## Screenshots, if any
Screenshot from 2020-05-27 09-57-21

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
@karantakalkar Please review

File: src/app/products/manage-tax-groups/view-tax-group/view-tax-group.component.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="mat-body-strong">
```
Comment (muskankhedia) [inline src/app/products/manage-tax-groups/view-tax-group/view-tax-group.component.html:16]: Only a minor change for a better UI, can you change the flex to 33%

File: src/app/products/manage-tax-groups/view-tax-group/view-tax-group.component.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          Name
+        </div>
+
+        <div fxFlex="50%">
```
Comment (muskankhedia) [inline src/app/products/manage-tax-groups/view-tax-group/view-tax-group.component.html:20]: And here the flex to 67%.

**Comment by BLasan** [comment]
@muskankhedia Done

**Comment by muskankhedia** [comment]
@BLasan Please rebase the commits so I can merge.

**Comment by BLasan** [comment]
@muskankhedia Done :)

**Pull request merged**